bridge-fungible-token/tests/harness.rs

```diff
@@ -447,6 +443,11 @@ mod success {
     #[tokio::test]
     async fn depositing_amount_too_small_registers_refund() {
         // In cases where BRIDGED_TOKEN_DECIMALS == PROXY_TOKEN_DECIMALS or BRIDGED_TOKEN_DECIMALS < PROXY_TOKEN_DECIMALS, this test will fail because it will attempt to bridge 0 coins which will always revert.
+        if BRIDGED_TOKEN_DECIMALS <= PROXY_TOKEN_DECIMALS {
+            return ();
+        }
```

Comment:
Should this be here or perhaps the setup should be adjusted? I'm not a fan of exiting a test early if something else has gone wrong

Reply:
So the testing situation is a little odd since the specific test scenarios are limited to what is in the forc.toml file when the contracts are compiled. Since there is no way to edit this in the middle of a test (that I know of) we currently have multiple tests for different setups. Some of the tests apply to all, but some (like this one) only applies to a specific config setup. I'm not sure how to better handle this other than maybe leave a TODO and add an issue to address this later?

Reply:
>Since there is no way to edit this in the middle of a test

Yeah this is still currently the case, but should be fixed soon with the introduction of the ability to set config-time consts via the SDK.

Reply:
I've created [an issue](https://github.com/FuelLabs/bridge-fungible-token/issues/49) to address this in the future once the sdk supports deploying contracts with different deploy time constants